### PR TITLE
Add keptn-lifecycle-toolkit on data/repositories.toml

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fixed issue #1742

Hi [agardnerIT](https://github.com/agardnerIT),
just trying to fix issue on file [data/repositories.toml](https://github.com/deepsourcelabs/good-first-issue/blob/8f9644b7c502789d1cfc9e9a06ebf38a3cc3e190/data/repositories.toml#L349C3-L349C28),
please check it out

#### ℹ️ Repository information

**The repository has**:

- [X] At least three issues with the `good first issue` label.
- [X] At least 10 contributors.
- [X] Detailed setup instructions for the project.
- [X] CONTRIBUTING.md
- [X] Actively maintained.

### CODE REVIEW

**Review by agardnerIT — APPROVED** [review]

**Comment by viktoriussuwandi** [comment]
Hi [agardnerIT](https://github.com/agardnerIT),
Is there anything else need to do to merge this PR ?
Please let me know

**Comment by agardnerIT** [comment]
> Is there anything else need to do to merge this PR ?

Not from my side. This repo isn't controlled by Keptn. Someone from deepsourcelabs needs to review.

**Comment by viktoriussuwandi** [comment]
> > Is there anything else need to do to merge this PR ?
> 
> Not from my side. This repo isn't controlled by Keptn. Someone from deepsourcelabs needs to review.

Maybe anyone can answer ?
@jai-deepsource @siddhant-deepsource @vishnu-deepsource @ashwin-deepsource

File: data/repositories.toml [hunk 1]
```diff
@@ -347,6 +347,7 @@ repositories = [
   'github.com/kariustobias/acme-rs',
   'github.com/KaTeX/KaTeX',
   'github.com/keptn/keptn',
+  'github.com/keptn/keptn-lifecycle-toolkit',
```
Comment (mowies) [inline data/repositories.toml:350]:
```suggestion
  'github.com/keptn/lifecycle-toolkit',
```

**Review by mowies — COMMENTED** [review]
fixed the repo link

**Comment by viktoriussuwandi** [comment]
Sorry, I have to close this PR because no action from deepsourcelabs side

**Pull request closed without merging**